notebooks/network_mapping/Supervised_Asset_Classification.ipynb

```diff
@@ -0,0 +1,1600 @@
+{
```

Comment:
<p>I think you could move this text up 2 blocks and include explanatory paragraphs for train, test, and predict functions.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/clx/pull/193/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='53'/>